auslib/web/admin/swagger/api.yaml

```diff
@@ -0,0 +1,441 @@
+swagger: '2.0'
+info:
+  version: '2.27'
+  title: "Balrog's Admin API"
+  termsOfService: "https://wiki.mozilla.org/Balrog"
+  license:
+    name: "Mozilla Public License 2.0"
+    url: "https://www.mozilla.org/en-US/MPL/2.0/"
+  contact:
+    name: "Ben Hearsum"
+    url: "https://github.com/mozilla/balrog"
+    email: "ben@hearsum.ca"
+  description: "Mozilla's Update Server. APIs for Balrog" #TODO write in detail using Github Flavored Markdown
+
+schemes:
+  - http
```

Comment:
_Definition from swagger docs_ : **schemes** : 
> The transfer protocol of the API. Values MUST be from the list: "http", "https", "ws", "wss". If the schemes is not included, the default scheme to be used is the one used to access the Swagger definition itself.

Will add https and test it out